### ISSUE TITLE
Rearrange help page

### ISSUE THREAD

**Issue by mray** [freedombox/Plinth#1150]
Looking at   https://github.com/freedombox/Plinth/pull/1149 I noticed:
From a UI perspective there are many problems with the current "help" section, that change even adds to that:

* The sidebar is not used consistent with other uses of that area.  
* One menu entry is not related to help at all (about)
* The most links are about the manual in one or the other way:
    One menu entry is the manual.
    One menu entry is the manual as pdf.
    One menu entry is the mentioning the manual and two other resources.

At this point we could remedy that a little bit by removing the sidebar and having just the huuuuuge help page an add one paragraph on top of it, containing links to a downloadable pdf version and the *other* resources: Wiki, Mailing-List & IRC (Matrix is missing there). I guess the link to "about" page can be put in the footer of the homepage. 

On top of that the manual itself is a HTML-dump of an intimidating amount of information with poor resolution screenshots. That section really needs some love.

Also the links on /help and /about to the wiki are broken (https://wiki.debian.org/FreedomBox)

**Comment by JosephKiranBabu** [comment]
@mray The links on /help and /about to the wiki (https://wiki.debian.org/FreedomBox) are not really broken. Did your browser timeout when trying to access them? The Debian wiki is very slow and might timeout sometimes.

**Comment by mray** [comment]
@JosephKiranBabu you are right. Funny, the links on those pages work again – might have been on my end.
